src/components/hint/index.tsx

```diff
@@ -397,31 +394,31 @@ class Hint extends Component<HintProps, HintState> {
     return tipPositionStyle;
   }
 
-  // renderOverlay() {
-  //   const {targetLayoutInWindow} = this.state;
-  //   const {onBackgroundPress} = this.props;
-  //   if (targetLayoutInWindow) {
-  //     const containerPosition = this.getContainerPosition();
-  //     return (
-  //       <View
-  //         style={[
-  //           styles.overlay,
-  //           {
-  //             top: containerPosition.top - targetLayoutInWindow.y,
-  //             left: containerPosition.left - targetLayoutInWindow.x,
-  //           },
-  //         ]}
-  //         pointerEvents="box-none"
-  //       >
-  //         {onBackgroundPress && (
-  //           <TouchableWithoutFeedback style={[StyleSheet.absoluteFillObject]} onPress={onBackgroundPress}>
-  //             <View flex />
-  //           </TouchableWithoutFeedback>
-  //         )}
-  //       </View>
-  //     );
-  //   }
-  // }
+  renderOverlay() {
+    const {targetLayoutInWindow = {y: 0, x: 0}} = this.state;
```

Comment:
Why are you setting a default value here? 
Also, it makes the check in line 400 redundant.. maybe it was intentional?

Reply:
Just because of the TS error, added a `ts-expect-error` for now

Reply:
In that case it's better to fix the actual issue. 
In this case `ts-expect-error` would never be solved, cause it's our code. If anything you can use `ts-ignore`
But I think that in this case we just fix it. 

You can declare the type of the `targetLayoutInWindow` in the state. 
You can use casting to keep it simple, something like that

```
  state = {
    targetLayoutInWindow: undefined as {x: number, y: number, width: number, height: number} | undefined,
    targetLayout: this.props.targetFrame,
    hintUnmounted: !this.props.visible
  };
```